bentoml/adapters/multi_image_input.py

```diff
@@ -36,64 +47,115 @@ class MultiImageInput(BaseInputAdapter):
             https://imageio.readthedocs.io/en/stable/format_png-pil.html
 
     Raises:
-        ImportError: imageio package is required to use ImageInput
+        ImportError: imageio package is required to use MultiImageInput
 
     Example usage:
-        ```
-        class MyService(BentoService):
-            @bentoml.api(input=MultiImageInput(input_names=('imageX', 'imageY')))
-            def predict(self, image_groups):
-                for image_group in image_groups:
-                    image_array_x = image_group['imageX']
-                    image_array_y = image_group['imageY']
-        ```
+
+    >>> from bentoml import BentoService
+    >>> import bentoml
+    >>>
+    >>> class MyService(BentoService):
+    >>>     @bentoml.api(input=MultiImageInput(input_names=('imageX', 'imageY')))
+    >>>     def predict(self, image_groups):
+    >>>         for image_group in image_groups:
+    >>>             image_array_x = image_group['imageX']
+    >>>             image_array_y = image_group['imageY']
```

Comment:
Could you also add example client code here to demo how to submit a request with multiple images to this endpoint?